server/routes/findInterventions/InterventionDetailsPresenter.test.ts

```diff
@@ -38,6 +39,30 @@ describe(InterventionDetailsPresenter, () => {
     })
   })
 
+  describe('tabs', () => {
+    const presenter = new InterventionDetailsPresenter(
+      interventionFactory.build({
+        serviceProvider: serviceProviderFactory.build({ name: 'Harmony Living' }),
+      })
+    )
+
+    it('returns an array of service provider fields to pass to the tab args, including a summary list', () => {
```

Comment:
I'm not sure what "to pass to the tab args" means here - are you referring to the `InterventionDetailsView.prototype.tabsArgs` method? If so, I don't think that a presenter would be expected to know about that method (view knows about presenter, not vice-versa).

How about just

```suggestion
    it('returns an array of summary lists, each with a title', () => {
```

Reply:
Yeah, really wasn't sure about this - have taken your suggestion